packages/driver/src/cy/snapshots.ts

```diff
@@ -99,15 +101,20 @@ export const create = ($$, state) => {
   }
 
   const getStyles = (snapshot) => {
-    const styleIds = snapshotsMap.get(snapshot)
+    const stylesOrStyleIds = snapshotsMap.get(snapshot)
```

Comment:
It feels weird to overload this object with different types, could we instead return an object like this?
```javascript
const styleObj = {
  ids: {
    headStyleIds,
    bodyStyleIds,
  },
  styles: {
    headStyles,
    bodyStyles,
  },
}
```

With that i think the code could be more easily read

```suggestion
    const {ids, styles} = snapshotsMap.get(snapshot)
    
    if (!ids || !styles) {
      return {}
    }
    
    if (styles) {
      return styles
     }
```